src/main/java/com/google/sps/classes/Utility.java

```diff
@@ -34,6 +34,9 @@
       "jdbc:mysql://localhost:3306/Mintern?useSSL=false&serverTimezone=America/Mexico_City";
   public static final String SQL_LOCAL_USER = "root";
   public static final String SQL_LOCAL_PASSWORD = "";
+
+  // Variables for user login status.
+  public static final int USER_LOGGED_OUT = -1;
```

Comment:
I know we already have some constants in Utility.java, but in reality, Utility.java should probably only contain utility methods, and Constants.java / SqlConstants.java / UserConstants.java should contain all the constants. Can you add a TODO to this effect? If you feel like it, you can move this (and any related) constants out into a Constants.java or UserConstants.java file, but I won't block on that for this PR.

Reply:
Added the TODO!